exchange.lua

```diff
@@ -1018,17 +1016,17 @@ end
 
 
 -- On success, returns true and a list of inbox entries.
--- On failure, returns false and an error message.
+-- On failure, an error message.
```

Comment:
It says that an error message is returned, which means it is returned. I haven't done it yet, but I wanted to make room for error-checking later on.